pkg/kubelet/cm/cpuset/cpuset_test.go

```diff
@@ -36,6 +36,11 @@ func TestCPUSetBuilder(t *testing.T) {
 	if len(elems) != result.Size() {
 		t.Fatalf("expected cpuset %s to have the same size as %v", result, elems)
 	}
+
+	b.Add(6)
+	if len(elems) != result.Size() {
```

Comment:
I would verify it with element than size for clarity
```
b.Add(newElement)
if result.Contains(newElement) {
  t.Fatalf("element should not be added to cpuset after result is called")
}
```

Reply:
We also have testify in k/k so this can be reduced to:

```
require.Contains(t, result, element, "Expected calls to Add after calling Result() to have no effect")
```

(https://pkg.go.dev/github.com/stretchr/testify/require#Contains)

Reply:
Thanks a lot for your feedback @adisky and @endocrimes! Applied your feedback in d1f25d7 but chose `require.False` over `require.Contains` to validate `CPUSet`'s `Contains` method, and lowercased `Expected` to be consistent with the other messages.